public/app/features/alerting/unified/utils/rules.ts

```diff
@@ -161,7 +161,8 @@ const alertStateToStateMap: Record<PromAlertingRuleState | GrafanaAlertState | A
   [AlertState.Paused]: 'warning',
   [AlertState.Alerting]: 'bad',
   [AlertState.OK]: 'good',
-  [AlertState.Pending]: 'warning',
+  // AlertState.Pending is not included because the 'pending' value is already covered by `PromAlertingRuleState.Pending`
+  // [AlertState.Pending]: 'warning',
```

Comment:
@grafana/alerting-frontend plz review this 🙏

Reply:
Hm they resolve to different string values though `pending` vs `Pending` – if this isn't a blocker for the upgrade to TS5 I think we should revert.

What's the type checker complaining about without this change?

Reply:
Do they?

 - `PromAlertingRuleState.Pending` has a value of `pending` ([source](https://github.com/grafana/grafana/blob/387c62d751078c9382ead9602b692b0d3679ed62/public/app/types/unified-alerting-dto.ts))
 - `GrafanaAlertState.Pending` has a value of `Pending` ([source](https://github.com/grafana/grafana/blob/387c62d751078c9382ead9602b692b0d3679ed62/public/app/types/unified-alerting-dto.ts/#L19))
 - `AlertState.Pending` has a value of `pending` ([source](https://github.com/grafana/grafana/blob/387c62d751078c9382ead9602b692b0d3679ed62/packages/grafana-data/src/types/alerts.ts/#L9))

Exact error is "An object literal cannot have multiple properties with the same name"

![image](https://github.com/grafana/grafana/assets/46142/922f6487-9c34-4438-a54b-7c427321add1)

Reply:
@gillesdemey politely making sure you get the ping for this ^^ 😌

Reply:
Sorry – I don't seem to get pinged for followup on these threads. You're quite right, I was comparing only `GrafanaAlertState` with `AlertState.Pending` :D 

Changes LGTM!